crates/rnote-cli/src/cli.rs

```diff
@@ -32,37 +34,66 @@ pub(crate) enum Commands {
         #[arg(short = 'i', long)]
         input_file: PathBuf,
         /// When importing a .xopp file, the import dpi can be specified.{n}
-        /// Else the default (96) is used.
-        #[arg(long)]
-        xopp_dpi: Option<f64>,
+        #[arg(long, default_value_t = XoppImportPrefs::default().dpi)]
+        xopp_dpi: f64,
     },
     /// Exports the Rnote file(s) and saves it in the desired format.{n}
-    /// When using --output-file, only one input file can be given.{n}
-    /// The export format is recognized from the file extension of the output file.{n}
-    /// When using --output-format, the same file name is used with the extension changed.{n}
-    /// --output-file and --output-format are mutually exclusive but one of them is required.{n}
-    /// Currently `.svg`, `.xopp` and `.pdf` are supported.{n}
-    /// Usages: {n}
-    /// rnote-cli export --output-file [filename.(svg|xopp|pdf)] [1 file]{n}
-    /// rnote-cli export --output-format [svg|xopp|pdf] [list of files]
+    /// See subcommands for usage
     Export {
         /// the rnote save file
+        #[arg(global = true)]
         rnote_files: Vec<PathBuf>,
-        /// the export output file. Only allows for one input file. Exclusive with output-format.
-        #[arg(short = 'o', long, conflicts_with("output_format"), required(true))]
-        output_file: Option<PathBuf>,
-        /// the export output format. Exclusive with output-file.
-        #[arg(short = 'f', long, conflicts_with("output_file"), required(true))]
-        output_format: Option<String>,
-        /// export with background
-        #[arg(short = 'b', long)]
-        with_background: Option<bool>,
-        /// export with background pattern
-        #[arg(short = 'p', long)]
-        with_pattern: Option<bool>,
+        /// The action that will be performed if the to be exported file(s) already exist(s)
+        #[arg(long, default_value = "ask", global = true)]
+        on_conflict: OnConflict,
+        /// export without background
+        #[arg(short = 'b', long = "no-background", action = ArgAction::SetFalse, global = true)]
+        background: bool,
```

Comment:
I am confused. Is this now supposed to be negated or not? Because it seems that you have changed it to non-negated after the review and discussion in Matrix, but missed updating the long-argument string.

Reply:
The Arg isn't changed, see the value given to no-background:  the variable is set to true by default and set to false when its provided. I think it makes the code a bit cleaner, but if you prefer the value being negated in the code as well, I can reverse that change.

Reply:
I see, but revert it please.